screens/ProfileScreen.tsx

```diff
@@ -1,23 +1,34 @@
 import * as React from 'react';
-import Icon from '../assets/svg-icons/settings.svg';
-import { StyleSheet } from 'react-native';
+import PlusIcon from '../assets/svg-icons/plus-circle.svg';
+import SettingsIcon from '../assets/svg-icons/settings.svg';
+import SearchIcon from '../assets/svg-icons/search.svg';
```

Comment:
Import like this can get messy when we have multiple svg files. It might be better if we have a separate file that serves as a directory for all svgs. It imports and exports all svg files like this:
```
import PlusIcon from '../assets/svg-icons/plus-circle.svg';
import SettingsIcon from '../assets/svg-icons/settings.svg';
import SearchIcon from '../assets/svg-icons/search.svg';

export {
  PlusIcon,
  SettingsIcon,
  SearchIcon
}
```

So in this file, you can do
```
import {
  PlusIcon,
  SettingsIcon,
  SearchIcon
} from '../SvgDirectory';
```
where `SvgDirectory` is the name of this directory file (feel free to name it something else, this is just an example).